File: m/rls/stend.r
file	stend - string end
include	rid:rider
include	rid:stdef

code	st_end - get string end

  func	st_end
	str : * char~			; string pointer
	()  : * char			; pointer to string terminator
  is	spin while *str++		; skip to end
	reply --str			; backup to terminator
  end


File: m/rls/rlb/stend.c
/* file -  stend - string end */
#include "m:\rid\rider.h"
#include "m:\rid\stdef.h"
/* code -  st_end - get string end */
char *st_end(
register char *str )
{ while ( *str++) {;}
  return ( --str);
} 
